Server/ServerController/route.py

```diff
@@ -0,0 +1,85 @@
+from Services.auth import Authentication
+from Services.adminServiceHandler import AdminServiceHandler
+from Services.chefServiceHandler import ChefServiceHandler
+from Services.employeeServiceHandler import EmployeeServiceHandler
+
+class Route:

```

Comment:
Class naming convention to follow `Controller` in the name

Reply:
I have renamed Route to RouteController